libass/Makefile_library.am

```diff
@@ -34,11 +34,16 @@ libass_libass_la_SOURCES = \
     libass/ass_outline.h libass/ass_outline.c \
     libass/ass_drawing.h libass/ass_drawing.c \
     libass/ass_bitmap.h libass/ass_bitmap.c libass/ass_blur.c \
-    libass/ass_rasterizer.h libass/ass_rasterizer.c libass/ass_rasterizer_c.c \
+    libass/ass_rasterizer.h libass/ass_rasterizer.c \
     libass/ass_render.h libass/ass_render.c libass/ass_render_api.c \
     libass/ass_bitmap_engine.h libass/ass_bitmap_engine.c \
+    libass/c/rasterizer.c \
+    libass/c/blend_bitmaps.c \
+    libass/c/be_blur.c \
+    libass/c/blur.c \
     libass/wyhash.h
 
+libass_libass_la_CPPFLAGS = -I$(top_srcdir)/libass
```

Comment:
Since now all programs/libraries have this set, how about we move this to toplevel `AM_CPPFLAGS` instead, dropping (these,) test, profile and compare’s `_CPPFLAGS` and change `fuzz_fuzz_CPPFLAGS` to `= $(AM_CPPFLAGS) $(FUZZ_CPPFLAGS)`?

Reply:
After that old warning came back. Looks like I should revert `c_` prefixes too.

Reply:
Done.